build.gradle

```diff
@@ -27,9 +27,37 @@ buildscript {
 repositories {
     mavenLocal()
     jcenter()
+    maven {
+        credentials {
+            username smartThingsArtifactoryUserName
+            password smartThingsArtifactoryPassword
+        }
+        url "http://artifactory.smartthings.com/libs-release-local"
```

Comment:
I'd like to not add more builds and artifacts into the deprecated Artifactory instance. In my opinion, migrating artifacts away from this instance is a mandatory prerequisite to this PR.